templates/squid.conf.header.erb

```diff
@@ -5,7 +5,7 @@
 cache_mem                     <%= @cache_mem %>
 
 <% if @memory_cache_shared -%>
```

Comment:
This is a problem if the parameter has been set to the boolean `false`.  Instead of `memory_cache_shared 'off'` you'll get nothing.